src/intellij/scala.ipr.SAMPLE

```diff
@@ -18,7 +18,8 @@
   </component>
   <component name="InspectionProjectProfileManager">
     <profile version="1.0">
-      <option name="myName" value="Project Default" />
+      <option value="Project Default" name="myName"/>
+      <inspection_tool enabled_by_default="false" level="WARNING" enabled="false" class="EmptyCheck"/>
```

Comment:
If it is quite unenabled, or even disabled, what does it mean to be not enabled at WARNING?